make_version.js

```diff
@@ -1,10 +1,10 @@
 var package = require('./package.json');
 var fs = require('fs');
 var date = new Date();
-today = date.getFullYear() + '-' + ('0' + (date.getMonth() + 1)).slice(-2) + '-' + ('0' + date.getDate()).slice(-2);
-version_string = '{"version":"' + package.version + '-' + today + '"}';
+var today = date.getFullYear() + '-' + ('0' + (date.getMonth() + 1)).slice(-2) + '-' + ('0' + date.getDate()).slice(-2)  + '-' + ('0' + date.getHours()).slice(-2) + '-' + ('0' + date.getMinutes()).slice(-2);

```

Comment:
as I mentioned before, I'd extract things into smaller variables:
```
var month = ('0' + (date.getMonth() + 1)).slice(-2);
...
```
I won't block PR on this though